src/components/report-suspicious-activity/tests/report-suspicious-activity-controller.test.ts

```diff
@@ -346,12 +365,32 @@ describe("report suspicious activity controller", () => {
       // Act
       await reportSuspiciousActivityConfirmationGet(
         req as Request,
-        res as Response
+        res as Response,
+        () => {}
       );
 
       // Assert
       expect(res.status).to.have.been.calledOnceWith(404);
       expect(res.render).to.have.been.calledOnceWith("common/errors/404.njk");
     });
+
+    it("Should call next when report parameters don't pass validation", async () => {
```

Comment:
Hi @liegeandlief - this test name is exactly the same as the own above - but it has a different req.query attribute, could we name them instead

"Should call next when the **event** query string parameter fails validation"
"Should call next when the **page** query string parameter fails validation"